sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SessionRecords/SparkBatchClientLiveTests/TestGetSparkBatchJobAsync.json

```diff
@@ -18,1720 +18,31 @@
-        ],
-        "x-ms-client-request-id": "a7b77865908d3d939f2e8d580fac06ba",
-        "x-ms-return-client-request-id": "true"
-      },
-      "RequestBody": null,
-      "StatusCode": 200,
-      "ResponseHeaders": {
-        "Content-Type": "application/json; charset=utf-8",
-        "Date": "Wed, 26 Aug 2020 05:54:33 GMT",
-        "Server": [
-          "Kestrel",
-          "Microsoft-HTTPAPI/2.0"
-        ],
-        "Strict-Transport-Security": "max-age=31536000; includeSubDomains",
-        "Transfer-Encoding": "chunked",
-        "x-ms-activity-id": "ff462885-4e04-4180-b376-4ce545fd6cfe",
-        "x-ms-client-request-id": [
-          "a7b77865908d3d939f2e8d580fac06ba",
-          "a7b77865908d3d939f2e8d580fac06ba"
-        ],
-        "x-ms-job-cluster": "http://171.3.39.9:8998/",
-        "x-ms-job-clusterrequested-on": "1/19/2020 6:44:14 AM \u002B00:00",
-        "x-ms-job-ended-on": "1/19/2020 6:47:34 AM \u002B00:00",
-        "x-ms-job-internal-id": "26",
-        "x-ms-job-livysubmission-on": "1/19/2020 6:46:05 AM \u002B00:00",
-        "x-ms-job-queued-on": "1/19/2020 6:46:03 AM \u002B00:00",
-        "x-ms-job-result": "Succeeded",
-        "x-ms-job-scheduled-on": "1/19/2020 6:44:13 AM \u002B00:00",
-        "x-ms-job-scheduler-state": "Ended",
-        "x-ms-job-submitted-by-name": "zhezhou@microsoft.com",
-        "x-ms-job-submitted-on": "1/19/2020 6:44:13 AM \u002B00:00",
-        "x-ms-job-type": "SparkServiceBatch",
-        "x-ms-request-id": "62c18ce4-fa0d-4476-85ec-d8d3d399435c",
-        "x-ms-response-time-ms": "28"
-      },
-      "ResponseBody": {
-        "id": 26,
-        "appId": "application_1579416350077_0001",
-        "appInfo": {
-          "driverLogUrl": "http://35c3cd39703446f7b5879bc0b8805f8c001de530649:8042/node/containerlogs/container_1579416350077_0001_01_000001/trusted-service-user",
-          "sparkUiUrl": "http://35c3cd39703446f7b5879bc0b8805f8c016c6327596:8088/proxy/application_1579416350077_0001/"
-        },
-        "state": "success",
-        "log": [
-          "\t tracking URL: http://35c3cd39703446f7b5879bc0b8805f8c016c6327596:8088/proxy/application_1579416350077_0001/",
-          "\t user: trusted-service-user",
-          "20/01/19 06:46:18 INFO ShutdownHookManager: Shutdown hook called",
-          "20/01/19 06:46:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-de3d608d-97b0-4ca5-a6c5-b05df4a40e8e",
-          "20/01/19 06:46:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-ab19f399-5768-4010-a5aa-0257fc7ccc71",
-          "20/01/19 06:46:18 INFO MetricsSystemImpl: Stopping azure-file-system metrics system...",
-          "20/01/19 06:46:18 INFO MetricsSystemImpl: azure-file-system metrics system stopped.",
-          "20/01/19 06:46:18 INFO MetricsSystemImpl: azure-file-system metrics system shutdown complete.",
-          "\nstderr: ",
-          "\nYARN Diagnostics: "
-        ]
+        "total": 0,
+        "sessions": []
```

Comment:
Is it expected that sessions is empty?

Reply:
Yes, this is from the TestGetSparkBatchJob test where we walk and print any outstanding sessions. It doesn't trigger any itself, so if there aren't any running nothing will trigger.

This is one of the tests I'm looking to improve in another PR.